web/js/students_of_class.js

```diff
@@ -230,7 +287,7 @@ function constructStudentTableRow(student) {
     let addSportResult = document.createElement("td");
     let addSportResultButton = document.createElement("span");
     addSportResultButton.onclick = () => {
-        document.getElementById("studentURL").value = studentURL;
+        document.querySelector("#studentURL").value = studentURL;
         createSportResultTable(studentURL, student);
         modalSportResult.modal('show');
         return false;
```

Comment:
Why does this onClick return anything ... ❓

Reply:
I dont know